domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt

```diff
@@ -67,13 +77,13 @@ class NumericInputEqualsRuleClassifierProviderTest {
   @Test
   fun testPositiveRealAnswer_positiveRealInput_valuesInRange_bothValuesMatch() {
     val inputs = mapOf(
-      "x" to createReal(
+      "x" to InteractionObjectTestBuilder.createReal(
```

Comment:
We should use variables like above for this one too.

Reply:
Can you elaborate please?

Reply:
Above where we are creating variable like
```kotlin
  private val POSITIVE_REAL_VALUE_3_5 = createReal(value = 3.5)	    InteractionObjectTestBuilder.createReal(value = 1.5)
```
create and use variables here. not directly calling the method and getting the value.

Reply:
@Victor-Titan Please use variables for this. 

```kotlin
 val inputs = mapOf(
      "x" to createReal(
        value = 5 * FLOAT_EQUALITY_INTERVAL
      )
    )
```

```kotlin
 private val FIVE_TIMES_FLOAT_EQUALITY_INTERVAL =
    InteractionObjectTestBuilder.createReal(value = 5 * FLOAT_EQUALITY_INTERVAL)

 val inputs = mapOf(
      "x" to FIVE_TIMES_FLOAT_EQUALITY_INTERVAL
    )
```

and Remove the below functions from `NumericInputEqualsRuleClassifierProviderTest` as we are going to use all the helper functions from the `InteractionObjectTestBuilder`. 

```kotlin
 private fun createReal(value: Double): InteractionObject {
    return InteractionObject.newBuilder().setReal(value).build()
  }

  private fun createString(value: String): InteractionObject {
    return InteractionObject.newBuilder().setNormalizedString(value).build()
  }
```


Ditto to all other files

Reply:
The test files have changed quite a bit, but there's still places where strings are directly used in the parameters, would you like me to create variables for them and use those instead?

Reply:
You can first update this branch with the latest develop and continue the work. As per your message in Gitter it looks like you branch is messed up and you are not able to update with the latest develop, in that case I suggest to create a new pr with  fresh branch.

Reply:
You are right, I seem to have made a mistake, I checked the main branch code and it hasn't changed the way it has for my local branch. Will rectify and do the needful.